src/AddressBond.jsx

```diff
@@ -15,34 +14,35 @@ class AddressBond extends InputBond {
 			bonds.me
 		], (reg, me) => ({
 			registry: isNullData(reg) ? null : reg,
-			internal: n == 'null' ? '0x0000000000000000000000000000000000000000' : n == 'me' ? me : null
+			internal: n === 'null' ? '0x0000000000000000000000000000000000000000' : n === 'me' ? me : null
 		}));
 	}
 
 	makeIcon (p) {
 		return p ? 'left' : this.state.ok
-				? (<i style={{opacity: 1}} className='icon'>
-					<AccountIcon
-						style={{opacity: 1, border: '0.5em solid transparent'}}
-						address={this.state.external}
-					/></i>)
-				: undefined;
+			? (<i style={ { opacity: 1 } } className='icon'>
+				<AccountIcon
+					style={ { opacity: 1, border: '0.5em solid transparent' } }
+					address={ this.state.external }
+				/></i>)
+			: undefined;
 	}
 
 	render () {
 		const labelStyle = {
 			position: 'absolute',
 			zIndex: this.props.labelZIndex || 10
 		};
+
 		return (
 			<div>
 				{InputBond.prototype.render.call(this)}
 				<div>
 					{this.state.ok
 						? ''
 						: this.state.extra.noChecksum
```

Comment:
Putting comment here, but there's a lot of instances where this should be changed.

`this.state.ok ? '' : ...` should be transformed to `!this.state.ok && ...`. It's useless to create a text node in the DOM with '', and it's much clearer.

Also, I saw a lot of `someVariable ? ... : null`, should be transformed to `someVariable && ...`

To be changed in other files too.